dbms/src/Client/Benchmark.cpp

```diff
@@ -76,6 +78,11 @@ class Benchmark
 		else
 			throw Exception("Unknown query processing stage: " + stage, ErrorCodes::BAD_ARGUMENTS);
 
+		if (!json_path.empty() && Poco::File(json_path).exists()) /// Clear file with previous results
```

Comment:
It is expected behavior. It allow to avoid reading data from previous test launches if current test is failed.